README.md

```diff
@@ -5,13 +5,16 @@ The `vm-virtio` workspace provides
 abstractions and implementation for the virtio queue and devices. For now, it
 consists of the following crates:
 
+- `virtio-bindings` -> provides Rust FFI bindings to the corresponding Linux kernel API;
```

Comment:
nit: we usually limit lines in md files to 80 chars.

Reply:
fixed